src/main/daml/Daml/Finance/Interface/Instrument/Bond/FloatingRate/Types.daml

```diff
@@ -5,6 +5,7 @@ module Daml.Finance.Interface.Instrument.Bond.FloatingRate.Types where
 
 import Daml.Finance.Interface.Types.Common.Types (InstrumentKey(..))
 import Daml.Finance.Interface.Types.Date.DayCount (DayCountConventionEnum)
+import Daml.Finance.Interface.Types.Date.FixingDates qualified as BondTypes (FloatingRate)
```

Comment:
No need to qualify the import here

Reply:
The interface type is also called `FloatingRate`, same as the instrument name:

```
-- | Describes the attributes representing a floating rate bond.
data FloatingRate = FloatingRate
  with
    instrument : InstrumentKey
      -- ^ The instrument's key.
```

I was thinking about maybe renaming that data type to `FloatingRateBond` or something, but then it would not follow the convention used for the other instruments.

Reply:
Makes sense. I would then suggest to qualify as `Types`, as it's slightly less confusing than `BondTypes`